src/odemis/driver/test/actuator_test.py

```diff
@@ -494,6 +506,10 @@ def test_move_abs(self):
         f.result()
         test.assert_pos_almost_equal(stage.position.value, {"x": 1e-06, "y": 2e-06})
         test.assert_pos_almost_equal(child.position.value, {"x": 1e-05, "y": 2e-05})
+        f = stage.moveAbs({"x": 1e-06})  # Test only move only one axis
+        f.result()
+        test.assert_pos_almost_equal(stage.position.value, {"x": 1e-06, "y": 2e-06})
+        test.assert_pos_almost_equal(child.position.value, {"x": 1e-05, "y": 2e-05})
```

Comment:
question: why are child positions one magnitude less compared to stage?

Reply:
Because we are testing with scale = 10